code/process_sequences.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Delong Tsway
+# dttsway@dons.usfca.edu
+# 2020-10-18
+
+# resources I consulted to build this code:
+# https://github.com/vsbuffalo/bioawk-tutorial/blob/master/README.md
+# https://opensource.com/article/17/2/command-line-tools-data-analysis-linux
+# https://www.poftut.com/awk-text-split-delimit-examples/
+# https://www.cyberciti.biz/faq/run-execute-sh-shell-script/
+# https://askubuntu.com/questions/420981/how-do-i-save-terminal-output-to-a-file
+
+FILE="/blast-db/sars-cov-2-seq-data/2020-10-15_ncbi_sars_cov_2_sequences.fasta.gz"
+
+if [ $# -eq 0 ]
+then
+ echo "There is no file path provided"
+exit 1
+fi
+
+if [ $# -gt 1 ]
+then
+echo "There is more than the one require argument provided"
+exit 1
+fi
+# echo statement to describe contents:
+echo "This is some information about the sequences from the Sars-Cov-2 database" > output/SeqSort.txt
+
+# code to count total number of sequences in file
+echo "Number of sequences in file:" >> output/SeqSort.txt
+bioawk -cfastx 'END{print NR}' $FILE | tee -a output/SeqSort.txt 
```

Comment:
Same as earlier. Remove the tee bit since you can just redirect the entire output at once when you run the script.